manifests/database/postgresql.pp

```diff
@@ -84,7 +84,7 @@
   }
 
   exec { 'update_pgpass':
-    command => "echo ${database_host}:5432:${database_name}:${database_user}:${database_password} >> /root/.pgpass",
+    command => "echo '${database_host}:5432:${database_name}:${database_user}:${database_password}' >> /root/.pgpass",
     path    => "/bin:/usr/bin:/usr/local/sbin:/usr/local/bin:${database_path}",
     unless  => "grep \"${database_host}:5432:${database_name}:${database_user}:${database_password}\" /root/.pgpass",
```

Comment:
this grep should be adjusted as well to add the single quotes to existing files?